tasks/tasks.py

```
@@ -5,61 +5,304 @@
+    """Task that imports all instance pools."""
+    def __init__(self, client_config, args):
+        super().__init__(name="import_instance_pools")
+        self.client_config = client_config
+        self.args = args
+
+    def run(self):
+        cl_c = ClustersClient(self.client_config)
+        cl_c.import_instance_pools()
+
+class JobsExportTask(AbstractTask):
+    """Task that exports all jobs.
+
+    The behavior is equivalent to `$ python export_db.py --jobs`, which lives in main function of
+    export_db.py.
+    """
+    def __init__(self, client_config, args):
+        super().__init__(name="export_jobs")
+        self.client_config = client_config
+        self.args = args
+
+    def run(self):
+        jobs_c = JobsClient(self.client_config)
+        jobs_c.log_job_configs()
+
+class JobsImportTask(AbstractTask):
+    """Task that imports all jobs.
+
+    The behavior is equivalent to `$ python import_db.py --jobs`, which lives in main function of
+    import_db.py.
+    """
+    def __init__(self, client_config, args):
+        super().__init__(name="import_jobs")
+        self.client_config = client_config
+        self.args = args
+
+    def run(self):
+        jobs_c = JobsClient(self.client_config)
+        jobs_c.import_job_configs()
+
+class MetastoreExportTask(AbstractTask):
+    """Task that exports all metastore tables.
+
+    The behavior is equivalent to `$ python export_db.py --metastore`, which lives in main function of
+    export_db.py.
+    """
+    def __init__(self, client_config, checkpoint_service, args):
+        super().__init__(name="export_metastore")
+        self.client_config = client_config
+        self.checkpoint_service = checkpoint_service
+        self.args = args
+
+    def run(self):
+        hive_c = HiveClient(self.client_config, self.checkpoint_service)
+        hive_c.export_hive_metastore(cluster_name=self.args.cluster_name, has_unicode=self.args.metastore_unicode)
```

Before passing in, should we just fail fast if args.cluster_name is not set?